lib/fog/aws/models/storage/file.rb

```diff
@@ -266,6 +268,7 @@ def save(options = {})
           options.merge!(metadata)
           options['x-amz-storage-class'] = storage_class if storage_class
           options['x-amz-tagging'] = tags if tags
+          options['x-amz-website-redirect-location' = website_redirect_location if website_redirect_location
```

Comment:
It looks like, from the tests, that this is missing the closing square bracket `]` just prior to the `=`. Could you please fix that up? Otherwise, looks good.

Reply:
Hi @geemus 
Thanks. Updated the file.